library/core/src/mem/maybe_uninit.rs

```diff
@@ -37,11 +41,11 @@ use crate::{fmt, intrinsics, ptr, slice};
 /// let b: bool = unsafe { MaybeUninit::uninit().assume_init() }; // undefined behavior! ⚠️
 /// ```
 ///
-/// Moreover, uninitialized memory is special in that it does not have a fixed value ("fixed"
-/// meaning "it won't change without being written to"). Reading the same uninitialized byte
-/// multiple times can give different results. This makes it undefined behavior to have
-/// uninitialized data in a variable even if that variable has an integer type, which otherwise can
-/// hold any *fixed* bit pattern:
+/// Moreover, uninitialized memory is special in that it does not have a fixed
+/// (unchanged unless written to) value . Reading the same uninitialized byte
+/// multiple times can give different results. This even makes it undefined
+/// behavior to have uninitialized data in a variable of integer type,
+/// which otherwise could hold any *fixed* bit pattern:
```

Comment:
For instance, the old paragraph here is entirely fine, whereas the new version is unnecessarily terse while also using the term "value" in questionable ways.

I think you are underestimating how much work it is to review changes to natural-language technical writing that is highly sensitive to its exact wording. Every single change you make needs to be clearly motivated or it's just not worth the effort.

Reply:
This one is definitely more opinion-based, and I definitely like to remove extraneous words, as it improves the signal-to-noise ratio. I believe this is useful when you want to slowly read something a few times, when trying to understand it. But I'll happily retract this one.

I'm not quite sure how to respond to how much work reviewing changes is. Yes, it is a lot of work. So is polishing existing text, especially when having to justify each change. I still think there is value in doing it, as there can be a lot of papercuts hiding in documentation.

Reply:
> I definitely like to remove extraneous words, as it improves the signal-to-noise ratio

Sorry, no, please don't. Just because you prefer super terse text where every single word counts, doesn't mean that everyone prefers that. You are likely making this text *less* readable for others with your proposed changes, so in 6 months we'll get the next PR by someone arguing for a slightly different style.

There's value in fixing papercuts, yes. There is no value in rewording based on subjective ideas about which way of saying the same thing is more clear, that's just useless churn -- and it will be endless churn as there'll always be the next PR arguing for a different wording.

Therefore, I'll only spend my time on changes that are clearly motivated by objective criteria.